src/asm/stages/stage_3.asm

```diff
@@ -0,0 +1,199 @@
+.section .boot, "awx"
+.intel_syntax noprefix
+.code32
+
+
+# This stage checks for 64 bit, Sets up paging and enters long mode
+
+stage_3:
+	mov ax, 0x10
```

Comment:
Please explain this magic with comments, or even better with a descriptive name for the const if possible

Reply:
fixed with comments